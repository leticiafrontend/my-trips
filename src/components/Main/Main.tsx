import { Container } from './styles'

export const Main = () => {
  return (
    <Container>
      <h1>My Trips</h1>
    </Container>
  )
}
