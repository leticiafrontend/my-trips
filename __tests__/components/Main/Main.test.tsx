import { render, screen } from '@testing-library/react'
import { Main } from '../../../src/components/Main'

describe('<Main />', () => {
  it('should render page', () => {
    const { container } = render(<Main />)

    expect(screen.getByText('My Trips')).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
