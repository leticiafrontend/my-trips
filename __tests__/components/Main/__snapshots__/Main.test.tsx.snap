// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render page 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 100vh;
  background-color: #1f1635ff;
  color: #fff;
}

.c0 h1 {
  font-size: 3.5rem;
}

<div
  class="c0"
>
  <h1>
    My Trips
  </h1>
</div>
`;
